### PR TITLE
[feature]: add generate doc cmd for supernode

### LINKED ISSUE #1097: [CI]  generating supernode cli doc with robot
## Why you need it?

Only dfget and dfdaemon will be generated cli docs under the dir of [docs/cli_reference](https://github.com/dragonflyoss/Dragonfly/tree/master/docs/cli_reference). And we should do that with supernode.

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnopg@gmail.com>

### Ⅱ. Does this pull request fix one issue?

fixes #1097

### CODE REVIEW

File: cmd/supernode/app/gen_doc.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func init() {
 	genDocCommand := &GenDocCommand{}
 	genDocCommand.cmd = &cobra.Command{
 		Use:           "gen-doc",
-		Short:         "Generate Document for dfget command line tool with MarkDown format",
+		Short:         "Generate Document for supernode command line tool with MarkDown format",
```
Comment (yunfeiyanggzq) [inline cmd/supernode/app/gen_doc.go]: ` with MarkDown format`  I think `with` isn't  appropriate,`in` can be better?

File: pkg/cmd/gen_doc.go [hunk 2]
```diff
@@ -32,11 +32,11 @@ type GenDocCommand struct {
 	path string
 }
 
-func init() {
+func NewGenDocCommand(name string) *cobra.Command {
 	genDocCommand := &GenDocCommand{}
 	genDocCommand.cmd = &cobra.Command{
 		Use:           "gen-doc",
-		Short:         "Generate Document for dfdaemon command line tool with MarkDown format",
+		Short:         fmt.Sprintf("Generate Document for %s command line tool with MarkDown format", name),
```
Comment (yunfeiyanggzq) [inline pkg/cmd/gen_doc.go]: ditto

File: cmd/supernode/app/gen_doc.go (hunk 1 above)
Comment (Starnop) [reply cmd/supernode/app/gen_doc.go]: Agree

**Comment by fengzixu** [comment]
LGTM

**Pull request merged**